src/Tickets/Commerce/Gateways/Stripe/REST/Publishable_Key_Endpoint.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace TEC\Tickets\Commerce\Gateways\Stripe\REST;
+
+use TEC\Tickets\Commerce\Gateways\Stripe\Merchant;
+use TEC\Tickets\Commerce\Gateways\Stripe\Refresh_Token;
+
+use TEC\Tickets\Commerce\Gateways\Stripe\Webhooks;
+use Tribe__Documentation__Swagger__Provider_Interface;
+
+use WP_REST_Request;
+use WP_REST_Server;
+
+/**
+ * Class Publishable_Key_Endpoint
+ *
+ * @todo Determine if this whole file is needed.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Commerce\Gateways\Stripe\REST
+ */
+class Publishable_Key_Endpoint implements Tribe__Documentation__Swagger__Provider_Interface {
+
+	/**
+	 * The REST API endpoint path.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected $path = '/commerce/stripe/key';
+
+	/**
+	 * Register the actual endpoint on WP Rest API.
+	 *
+	 * @since TBD
+	 */
+	public function register() {
+		$namespace     = tribe( 'tickets.rest-v1.main' )->get_events_route_namespace();
+		$documentation = tribe( 'tickets.rest-v1.endpoints.documentation' );
+
+		register_rest_route(
+			$namespace,
+			$this->get_endpoint_path(),
```

Comment:
I dont think we need this file anymore. @moraleida confirm?

Reply:
Correct, this file was removed in my branch